helm/nats-operator/README.md

```diff
@@ -0,0 +1,139 @@
+| `auth.username`                      | Client authentication username                                                               | `true`                                          |
+| `auth.password`                      | Client authentication password                                                               | `true`                                          |
+| `auth.users`                         | Allows multi-user authentication of 2 or more user                                           | `[]`                                            |
+| `tls.enabled`                        | Enable TLS                                                                                   | `false`                                         |
+| `tls.serverSecret`                   | Certificates to secure the NATS client connections (type: kubernetes.io/tls)                 | `nil`                                           |
+| `tls.routesSecret`                   | Certificates to secure the routes. (type: kubernetes.io/tls)                                 | `nil`                                           |
+| `clusterSize`                        | Number of NATS nodes                                                                         | `3`                                             |
+| `configReload.enabled`               | Enable configuration reload                                                                  | `false`                                         |
+| `configReload.registry`              | Reload configuration name                                                                    | `docker.io`                                     |
+| `configReload.repository`            | Reload configuration image name                                                              | `connecteverything/nats-server-config-reloader` |
+| `configReload.tag`                   | Reload configuration image tag                                                               | `0.2.2-v1alpha2`                                |
+
+### Example
+
+Here is an example of how to setup a NATS cluster with client authentication.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. 
+
+```bash
+$ helm install --name my-release --set auth.enabled=true,auth.username=my-user,auth.password=T0pS3cr3t .
+```
+
+You can also specify more than 1 user using this way:
+
+```bash
+helm install --name my-release --set auth.enabled=true,auth.users[0]=my-user,auth.users[0].password=T0pS3cr3t,auth.users[1]=my-user-2,auth.users[1].password=MyS3cr3tP4ssw0rd .
+```
+You can consider editing the default values.yaml as it is easier to manage:
+
+```yaml
+...
+auth:
+  enabled: true
+
+  #username: 
+  #password: 
+
+  # values.yaml
+  users:
+    - username: "my-user"
+      password: "T0pS3cr3t"
+    - username: "my-user-2"
+      password: "MyS3cr3tP4ssw0rd"
```

Comment:
we could also declare `permissions:` here but could be done in a different PR

Reply:
Yeah of course! I'll remind myself to create an issue after this one is merged.

Reply:
Issue created https://github.com/nats-io/nats-operator/issues/67